pkg/preparer/listener.go

```diff
@@ -124,6 +124,8 @@ func (l *HookListener) Sync(quit <-chan struct{}, errCh chan<- error) {
 				sub.WithField("err", err).Errorln("Could not write hook link")
 			} else {
 				sub.NoFields().Infoln("Updated hook")
+				// Send back a nil error to signal success!
+				errCh <- nil
```

Comment:
I think I'm for this. There's a related thing I want to talk about. It's related to making this happen, but I would also need to make the change in either case.

Looks like I need to make a change to `WatchPods`. `WatchPods` right now sends errors on its error channel and new manifests on its manifest channel. I think I need to add a third channel on which it sends something if it wants to say "Hi, I read from consul without errors but I have no new manifests"